marketdata/rest.go

```diff
@@ -766,9 +777,47 @@ func (c *Client) GetCryptoTrades(symbol string, req GetCryptoTradesRequest) ([]C
 	return resp[symbol], nil
 }
 
-// GetMultiTrades returns trades for the given crypto symbols.
+// GetCryptoPerpsTrades returns the trades for the given crypto perpetual futures symbol.
+func (c *Client) GetCryptoPerpsTrades(symbol string, req GetCryptoTradesRequest) ([]CryptoPerpsTrade, error) {
```

Comment:
Historical trades / quotes / bars are not supported by the server currently, so they should not be added.

Reply:
Removed.